src/components/dashboard/Claim.js

```diff
@@ -88,6 +97,15 @@ const Claim = props => {
     await goodWallet
       .checkEntitlement()
       .then(entitlement => setState(prev => ({ ...prev, entitlement: entitlement.toNumber() })))
+      .catch(e => {
+        log.error('gatherStats failed', e.message, e)
+        showErrorDialog('Something unexpected happened', '', {
+          boldMessage: 'Try again.',
```

Comment:
@AlexeyKosinski same here
i've asked you review all error messages in this issue

Reply:
@StanislavShevchenko please change this error message to: "The error happens in blockchain request, please try again later"